test/cli/workspace/create/args

```diff
@@ -1 +1 @@
-remote workspace create --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% --name foo --type snowflake --size small
+remote workspace create --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% --name foo --type snowflake --size small
```

Comment:
Tu je chyba, default host je `connection.keboola.com`, lenze testovacie projekty mame aj na inych stacks.
Mozno by tam nemal byt ziadny default host :thinking: pri niektorych prikazoch (kde potrebujes iba komponenty) to moze davat zmysel, inde nie